recipes/zstash/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "zstash" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/zstash/archive/v{{ version }}.tar.gz
+  sha256: 16d4169dcf49057ac86fb78410bffbc9ceba7680f73d4f88659992c18376cb63
+  patches: 
+    - patches/fix-setup-py.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+
+  run:
+    - python >=3.6
+    - six
+    - globus-sdk >=2
+    - fair-research-login
+
+test:
+  imports:
+    - zstash
+  commands:
+    - zstash --help
+    # will reenable after E3SM-Project/zstash#250
+    - pip check
+    - ${PYTHON} -m unittest tests/test_*.py
```

Comment:
@mahf708, it looks like the zstash unit testing is only useful on HPC. @forsyth2, is that correct?

Reply:
Yes, `zstash` and `zppy` are only officially supported on the E3SM-Unified machines. (Development and testing occur on Chrysalis, Perlmutter, and Compy). The `zstash` unit tests in particular are best run on Perlmutter as the other machines don't have the HPSS access.

Reply:
Okay, @mahf708, please try commenting out the unit tests for `zstash` here.  Thanks @forsyth2.